script/room.js

```diff
@@ -797,6 +797,7 @@ var Room = {
 				$SM.set('stores["'+k+'"]', 0);
 			}
 			
+			k = _(k);
```

Comment:
I don't think that one was being used for lookups, was it? If so, I should
have caught it too...

On Thu, Apr 30, 2015 at 10:48 AM Andrea Rendine <notifications@github.com>
wrote:

> In script/room.js
> <https://github.com/doublespeakgames/adarkroom/pull/345#discussion_r29435743>
> :
>
> > @@ -797,6 +797,7 @@ var Room = {
> >  				$SM.set('stores["'+k+'"]', 0);
> >  			}
> >  			
> > +			k = _(k);
>
> I will fix it right now.
> However, I did the same for outside.js (in that case it's a variable
> called "name"). I saw that you merged my change, but perhaps I should have
> used something different there, too, for consistency. I'll give it a try.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/doublespeakgames/adarkroom/pull/345/files#r29435743>.
>